assets/css/main.css

```diff
@@ -325,10 +334,38 @@ html, body {
     border-color: #bce8f1;
 }
 
+@media (min-width: 1200px) and (max-width: 1800px) {
+    .frontpage .card {
+        width: 23%;
+    }
+}
+
+@media (min-width: 960px) and (max-width: 1200px) {
+    .frontpage .card {
+        width: 30%;
+    }
+}
+
+@media (min-width: 670px) and (max-width: 960px) {
+    .frontpage .card {
+        width: 47%;
+    }
+}
+
+@media (max-width: 670px) {
+    .frontpage .card {
+        width: 100%;
+    }
+}
+
 .frontpage a:hover {
     text-decoration: none;
 }
 
+.frontpage .card-clickable {
+    flex-grow: 1;
```

Comment:
What happens if this isn't applied?

Reply:
This makes sure that the footer will stick to the bottom of the card, and the clickable part is the one that grows, if there is need for the cards to vertically grow.

Reply:
Ah, interesting! Thanks